File: Java/dependency-injection/_02_control-freak/src/main/java/com/github/grzesiek_galezowski/DependencyInjection/_1_ControlFreak/Inbound/UdpSocket.java
package com.github.grzesiek_galezowski.DependencyInjection._1_ControlFreak.Inbound;

import java.util.Random;

class UdpSocket {
  public boolean receive(byte[] frameData) {
    new Random().nextBytes(frameData);
    return true;
  }
}

File: CSharp/DependencyInjection/DependencyInjection/DependencyInjection/_1_ControlFreak/Inbound/UdpSocket.cs
using System;

namespace DependencyInjectionBefore._1_ControlFreak.Inbound
{
  class UdpSocket
  {
    public bool Receive(out byte[] frameData)
    {
      frameData = new byte[100];
      new Random().NextBytes(frameData);
      return true;
    }
  }
}